### ISSUE TITLE
Improve outgoing communication

### ISSUE THREAD

**Issue by AlexLakatos** [mozilla/Reps#73]
## Goal:

Improve outgoing communication done by the Reps Leadership: Council, Peers and MO
## Roles:

Responsible: @alexlakatos 

Accountable: Reps Council

Supporting: @brianking, @MichaelKohler

Consulted: @couci 

Informed: Reps
## Required:
- [x] Create a draft survey about communication channel preferences for Reps (@alexlakatos @MichaelKohler , Due: Sep 27th) 
- [x] Present draft at the Council meeting and get it reviewed (@MichaelKohler, Due: Oct 5th) 
- [x] Share survey with Reps (Reps Council, Due: Oct 28th)
- [x] Gather survey results and generate a report (@alexlakatos @MichaelKohler , Due: Nov 12th)
- [x] Test most common result from the survey (@alexlakatos , Due: Nov 30th)

**Comment by nukeador** [comment]
Moving back to backlog until we have a discussion about this at Council level.

**Comment by MichaelKohler** [comment]
As discussed in the meeting today, Alex and I will share a proposal doc to work on and then present our ideas on how to reach out and what exactly what we need to find out with that in the next meeting for refinement.

**Comment by hmitsch** [comment]
I don't think this needs a council vote. Personally I'd like to vomit reading this. As my professional coach says: If it hurts it's a strong indicator that you are on the right track. 

Go for it. I love you guys!

**Comment by nukeador** [comment]
Nobody suggested Council vote, but council review/input on the plan.

**Comment by MichaelKohler** [comment]
Alex (mostly) and I (supporting) have come up with a draft document to explain the problem statement, what we want to find out, possible questions for a survey and next steps after the survey here: https://docs.google.com/document/d/1mZ_liU4yw_ug6FVs7_Ga9GSwRx06RCnRqto7TtA1uJg/edit?usp=sharing

This is not 100% done, so we kindly ask all Council members to raise any questions or feedback in the doc :)

**Comment by MichaelKohler** [comment]
Problem statement in the doc is updated. We will talk about this today in the Council meeting.

**Comment by hmitsch** [comment]
Have I already mentioned how much I love this initiative, @alexlakatos and @michaelkohler?!

Short feedback. Please reach out in case clarification is needed (and please tag me and ping me on Telegram if possible):

* We need to get Sean Rich in copy to this bug and documents. He is owning CoCo (C24N, Communication & Collaboration) at Mozilla IT and together with me we are looking at "CoCo 2017". I will make sure to get broader participation in this conversation as time passes and we progress. Our initiative is very much in its infancy. 
I will take care of that and you will have to accept his document access requests. 

* In addition to the qualitative research (survey), you suggest I'd like to propose taking a quantitative path as well (analytics, log analysis). You know how people react when you ask them something: they often report desires instead of facts. As Mitchell says "data is the new oxygen". 

Keep it up, lovin' you!

**Comment by MichaelKohler** [comment]
@AlexLakatos could you please review the survey draft? I've added the questions from the doc. Once we have your initial review we can send it out for further feedback.

**Comment by MichaelKohler** [comment]
Communication channels survey has been sent to Council for final review. Since we have another survey to be sent out early next week that is more time critical, we will delay sending this one by around 10 days.

**Comment by MichaelKohler** [comment]
Alex will send out the survey end of the week since the Budget survey is done now. I've updated the timeline in comment 0 to reflect the new timeline.

**Comment by MichaelKohler** [comment]
The survey is still open and we will leave it open for a few more days since we would like to have as much diverse information on channels as possible. Further it will also go out on other channels we haven't announced it yet so we have a broad existing channels spectrum to make sure that all Reps have seen this survey.

**Comment by MichaelKohler** [comment]
Result has been published here: https://discourse.mozilla-community.org/t/communication-channels-survey/11943/5

Alex and I will work on next steps + a proposal for content.

**Comment by MichaelKohler** [comment]
Currently we are working on the visualization part so it's easily understandable. Let's create a new issue for the content exploration once we've shared the graphics.

**Comment by MichaelKohler** [comment]
As per latest voting, we have agreed to the following:

* Create a list of all channels that we are using so people can find them easily
* Focus on reps-general and Discourse moving forward (with the end goal to only have one channel that is synchronized)
* Pin Discourse thread on Telegram when they happen
* Create a newsletter - at least monthly
* We will still have the external communication channels: Twitter, Blog

I will create the list of channels to use and it's current purpose tomorrow. I will also announce this to all Reps. Once this is done, we can get into the content part (will create a new issue) and we might need to adjust the purpose after that, but we won't know until we have figured out all the content parts and strategy.

**Comment by MichaelKohler** [comment]
Documentation started here: https://wiki.mozilla.org/ReMo/Communication

**Comment by MichaelKohler** [comment]
Last open questions from my side:
* What should we do with the IRC Channels? (#remo, #remo-dev)
* What should we do with the reps-dev mailing list? We have the Discourse for that and it's rarely used. Kill it?

--> Started a thread on the Council alias to discuss this

**Comment by ioana-chiorean** [comment]
Follow up issue for newsletter. I think we need to think this true as it will require a bit of work. 
https://github.com/mozilla/Reps/issues/119

**Comment by MichaelKohler** [comment]
Community Development Team issue for it: https://github.com/mozilla/community-development/issues/496

**Comment by MichaelKohler** [comment]
Initial improvements are done, we will create a new issue with follow tasks once necessary.
